engine/runtime-compiler/src/main/scala/org/enso/compiler/data/CompilerConfig.scala

```diff
@@ -19,7 +20,7 @@ case class CompilerConfig(
   warningsEnabled: Boolean            = true,
   privateCheckEnabled: Boolean        = true,
   staticTypeInferenceEnabled: Boolean = false,
-  dumpIrs: Boolean                    = false,
+  irDumper: Option[String]            = None,
```

Comment:
```suggestion
  private val irDumper: Option[String]            = None,
```
and then have a getter:
```scala
def getDumper(): Option[IRDumper]
```